charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -741,3 +741,16 @@ servicebus:
         subscriptionNeeded: yes
     queues:
       - name: tribunals-to-hearing-api
+
+wa:
```

Comment:
No point duplicating as most of it is default in https://github.com/hmcts/sscs-tribunals-case-api/blob/9762d8335f8300031e53a8332e0b485005e07998/charts/sscs-tribunals-api/values.yaml#L144-L151

Reply:
Yes, I will remove this.